src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -50,23 +48,20 @@ The last step here is to make sure that the current sync process has completed:
 
 A series of tools and helpers to run migrations is available in [this repository](https://github.com/flodolo/fluent-migrations):
 * Clone the repository locally and follow the instructions available in the [README](https://github.com/flodolo/fluent-migrations/blob/master/README.md) to set the configuration file.
-* Add the new migration recipes in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder, so it’s possible to run multiple migrations in one pass. More information about the folder’s structure are available in the repository’s README.
+* Add the new migration recipes in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder, so it’s possible to run multiple migrations in one pass. More information about the folder’s structure is available in the repository’s README.
 
 At this point, while each migration has been tested as part of the review before landing, it’s always good to run the migration against one locale, e.g. `it`:
-* Update the locale’s repository, check the results of `compare-locales`. Assuming l10n repositories are cloned in `~/migrations/locales`, and the quarantine repository in `~/migrations/gecko-strings-quarantine`, the command to run is `compare-locales ~/migrations/gecko-strings-quarantine/_configs/browser.toml ~/migrations/locales it`. Save the output in order to compare it with the results after the migration.
+* Update the locale’s repository, check the results of `compare-locales`. Assuming the l10n repository is cloned in `~/migrations/firefox-l10n`, and the quarantine repository is in `~/migrations/firefox-l10n-source` and the `update` branch is checked out, the command to run is `compare-locales ~/migrations/firefox-l10n-source/_configs/browser.toml ~/migrations/firefox-l10n it`. Save the output in order to compare it with the results after the migration.
 * Run the migration, without pushing, only for `it`: `./scripts/migrate it wet-run`. Then run compare-locales again, and check if the results are as expected: migrated strings should not appear as missing anymore, and there should be no errors.
-* If everything looks as expected, re-run the script adding the push option `./scripts/migrate it wet-run push`.
```

Comment:
Why removing this? If something goes wrong in the next step, you'll have a pending commit locally.

Reply:
I was trying to reduce the amount of testing steps and simplify instructions, this seemed redundant since it'd be run again when you run all (and running everything doesn't take as much time anymore). I'm not quite sure what the problem with a pending local commit is, if the step failed here there'd still be a pending commit either way.

Reply:
Right, I think we can get away without this step, especially considering that the default is running migrations in a branch. If you were running against main, you risk having to pull and rebase, but this was much more disruptive on hg (there's no "pull and rebase", there's "pull+merge+commit+push" and you might not notice the issue when running 100+ locales).